lib/TInitialState.php

```diff
@@ -98,17 +98,23 @@ protected function publishInitialStateForUser(IUser $user, IRootFolder $rootFold
 		);
 
 		$attachmentFolder = $this->talkConfig->getAttachmentFolder($user->getUID());
+		$freeSpace = 0;
 
 		if ($attachmentFolder) {
 			try {
 				$userFolder = $rootFolder->getUserFolder($user->getUID());
 
-				if (!$userFolder->nodeExists($attachmentFolder)) {
-					$userFolder->newFolder($attachmentFolder);
+				try {
```

Comment:
is this expensive (query and loadwise) because if so we might just want to do it for guest app users and assume a properly configured normal case

Reply:
Not sure if expensive, usually I don't think so. This is also queried on every propfind in the files app.

"normal case" can vary: there are use cases where regular users are created with zero quota outside of the guests app (for those who don't want to use the app). In the latter use case users could also be told to only work on an external storage that is mounted for everyone.